admin_guide/overcommit.adoc

```diff
@@ -330,35 +330,34 @@ required.
 [[disabling-swap-memory]]
 === Disabling Swap Memory
 
-It is important to understand that oversubscribing the physical resources on a
-node affects resource guarantees the Kubernetes scheduler makes during pod
-placement. For example, suppose two guaranteed pods have reached their memory
-limit. Each container may start using swap. Eventually, if there is not enough
-swap space, processes in the pods can be terminated (due to the system being
-oversubscribed).
+You can disable swap by default on your nodes in order to preserve quality of
+service guarantees. Otherwise, physical resources on a node can oversubscribe,
+affecting the resource guarantees the Kubernetes scheduler makes during pod
+placement. 
```

Comment:
@derekwaynecarr This seems odd. It's as though having it enabled is a bad thing. Is there a reason why it's not just disabled by default instead of the user needing to do this??